src/explore-education-statistics-frontend/src/modules/find-statistics/FindStatisticsPage.tsx

```diff
@@ -166,7 +165,7 @@ const FindStatisticsPage: NextPage = () => {
     >
       <InsetText>
         <h2 className="govuk-heading-m">This page has changed</h2>
-        <p className={classNames('govuk-body', 'govuk-warning-text')}>
+        <p className="govuk-body">
```

Comment:
This doesn't fix the problem. The issue is that we're using a heading here incorrectly.

We should do this instead:
```tsx
<p className="govuk-heading-m">This page has changed.<p>
<p>...</p>
```

Note that we don't need `govuk-body` on `p` elements typically as that styling is applied by default.